common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -254,4 +254,8 @@ object ConfigKeys {
 
   val transactions = "whisk.transactions"
   val stride = s"$transactions.stride"
+  val invokerDocker = "whisk.docker"
```

Comment:
this is duplicating https://github.com/apache/incubator-openwhisk/blob/master/common/scala/src/main/scala/whisk/core/WhiskConfig.scala#L250

Reply:
yup.